nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -286,6 +295,25 @@ private static LibraryModels loadLibraryModels() {
             .add(methodRef("android.webkit.WebView", "getUrl()"))
             .build();
 
+    private static final ImmutableSet<MethodRef> NONNULL_RETURNS =
```

Comment:
Do we want to make in this decision for everyone using NullAway?  Or should we leave it to plugin library models to decide?

Reply:
Ok, sounds like a reasonable decision.  I only ask because these models are explicitly contradicting annotations in the code, so in some sense we are contradicting the intention of the library authors here.  But still, usability out of the box is of paramount importance.